tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -153,12 +153,13 @@ public void importGraphDef(byte[] graphDef, String prefix) throws IllegalArgumen
    * If {@code dx} is null, the implementation will use dx of {@link org.tensorflow.op.core.OnesLike OnesLike} for all
    * shapes in {@code y}.
    * 
+   * @param scopeName name of the subscope into which gradients operations are added. If null, defaults to "gradients".
```

Comment:
Perhaps call this "prefix" and say: "String prefix applied to the names of nodes added to the graph to compute gradients" since the notion of a "scope" or "subscope" isn't part of these low level methods.